src/components/Chip/Chip.tsx

```diff
@@ -11,17 +12,28 @@ import { StyledChip, StyledDeletableChip, StyledLabel } from "./Chip.styled";
 const Chip = (props: ChipsProps & HTMLAttributes<HTMLDivElement>): ReactElement => {
 	const { label, required, disabled, selected, onDelete, onClick } = props;
 
+	const [deletableChipWidth, setDeletableChipWidth] = useState(0);
+
+	const deletableChipRef = useRef(null);
+
+	useEffect(() => {
+		setDeletableChipWidth(deletableChipRef?.current?.offsetWidth);
+	}, [onDelete, label])
+
 	return onDelete ? (
-		<StyledDeletableChip
-			label={<StyledLabel>{label}</StyledLabel>}
-			{...props}
-			required={required}
-			disabled={disabled}
-			selected={selected}
-			deleteIcon={<ClearIcon data-testid='delete-icon-test-id' />}
-			onDelete={onDelete}
-			data-testid="delete-chip-testid"
-		/>
+		<Tooltip text={deletableChipWidth >= 186 ? label : ""}>

```

Comment:
This works only for fixed-width chips (for example in chip playground / kitchen sink) but doesn't for the advanced selection once you start adding more because then the scrollbar appears and makes the chips get smaller by a few pixels making the text still show the "..." but not reaching the 186 px. I feel we could just keep the tooltip to always render when using deletable chips.